### PR TITLE
docs: adds installation instructions for the package

### PR DESCRIPTION
**Pull request by baywet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,16 @@ An "alpha" version of a generated .NET SDK in C# from [GitHub's OpenAPI spec](ht
 
 ## How do I use it?
 
+### Installation
+
+To install the package, you can use either of the following option :
```
Comment (IEvangelist) [inline README.md:11]:
```suggestion
To install the package, you can use either of the following options:
```

File: README.md [hunk 2]
```diff
@@ -6,6 +6,16 @@ An "alpha" version of a generated .NET SDK in C# from [GitHub's OpenAPI spec](ht
...
+
+- In Visual Studio, from the Package Explorer, search for `Octokit.NET.SDK`, or
+- Type `Install-Package Octokit.NET.SDK` into the Package Manager Console, or
+- Type `dotnet add PathToCSProj package Octokit.NET.SDK` in a terminal (replace `PathToCSProj` by the path to the csproj file you want to add the dependency)
```
Comment (IEvangelist) [inline README.md:15]:
```suggestion
- Type `dotnet add ./path/to/myproject.csproj package Octokit.NET.SDK` in a terminal (replace `./path/to/myproject.csproj` by the path to the _*.csproj_ file you want to add the dependency)
```

**Review by IEvangelist — COMMENTED** [review]
LGTM, made a few minor suggestions.

**Comment by baywet** [comment]
@IEvangelist thanks for the suggestions! added.

**Review by nickfloyd — APPROVED** [review]
Thanks @baywet ❤️ !

**Pull request merged**